app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return 4 + self._hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self._nickname}. "
+                f"{self._nickname} has a hummer "
+                f"of the {self._hummer_level} level")
```

Comment:
Consider using double quotes for consistency in your code style, as per the checklist recommendation.